test/CheckedCRewriter/const_tests.c

```diff
@@ -0,0 +1,33 @@
+// Tests for Checked C rewriter tool.
+//
+// Checks properties of functions.
+//
+// RUN: checked-c-convert %s -- | FileCheck -match-full-lines %s
+// RUN: checked-c-convert %s -- | %clang_cc1 -verify -fcheckedc-extension -x c -
+// expected-no-diagnostics
+
```

Comment:
Other type qualifiers would need to be supported independently. There's an enum that I populate for this, I'd need to add more cases and move information from the `QualType` into that enum. I can make that change for other type qualifiers as part of this PR.